NFR.md

```diff
@@ -209,22 +211,23 @@ the system such as logs written by SLF4J are allowed.
 7.1.2 The component is allowed to send e-mails only through an
 external mail agent (for example, postfix).
 
-7.1.3 All distributed components of the application MUST be packaged as
-both deb and rpm files. Source code MUST be supplemented with scripts
-required for building deb and rpm packages.
+7.1.3 All distributed components of the application MUST be packaged for the 
+platforms defined in section 3.5. Source code MUST be supplemented with 
+scripts required to build packages for the supported platforms.

```

Comment:
This place is a bit tricky, since for Docker we decide what is included in the containers, so we couldn't say that all distributed packages are necessarily packaged for Docker. E.g. currently the HWtoken addon isn't.

So maybe it would be good to say here something in the lines of:
```suggestion
7.1.3 All distributed components of the application MUST have native packaged for the 
platforms defined in section 3.5. Source code MUST be supplemented with 
scripts required to build native packages for the supported platforms. Docker instead
is a special case and packages included in the containers are determined by the maintainers
on a case-by-case basis.
```